docs/content/en/docs/tutorials/developer-journey.md

```diff
@@ -0,0 +1,159 @@
+
+For a guided Cloud Shell tutorial on how a developer's journey might look in adopting Skaffold, follow:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleContainerTools/skaffold/docs/content/en/docs/tutorials&cloudshell_workspace=.&cloudshell_tutorial=developer-journey.md)
+
+### What you'll learn
+
+* How to develop on minikube locally and move to an enterprise managed kubernetes cluster with ease
+* How to easily switch buildpack versions to comply with security demands
+
+___
+
+**Time to complete**: <walkthrough-tutorial-duration duration=15></walkthrough-tutorial-duration>
+Click the **Start** button to move to the next step.
+
+## First steps
+
+### Prepare the Environment
+Create a project called `skaffold-tutorial` with a valid billing account in the Google Cloud [Manage Resources Console](https://console.cloud.google.com/cloud-resource-manager)
+
+Run the start script to deploy a GKE cluster we will use later and install the latest version of Skaffold. Although Skaffold is already installed on cloud shell, we'll install the latest version for good measure.
+```bash
+chmod +x start.sh && ./start.sh
+```
+
+### Start a Minikube cluster
+
+We'll use `minikube` as our local kubernetes cluster of choice.
+
+Run:
+```bash
+minikube start
+```
+
+## Run the App to Minikube Using Skaffold
+
+### Deploy the App
+Run:
+```bash
+skaffold dev
+```
+
+**Important:** note the software versions under the `DETECTING` phase of the buildpack output. These will be important later.
+
+Open a second terminal. In order to connect to the application to make sure its working, you need to start the minikube load balancer by running:
+```bash
+minikube tunnel
+```
+
+Open a third terminal. Find out the load balancer IP by recording the external IP as `EXTERNAL_IP` after running:
+```bash
+kubectl get service
+```
+
+Ensure the app is responding by running:
+```bash
+curl http://EXTERNAL_IP:8080
+```
+
+### Change the App and Trigger a Redeploy
+
+In the Cloud Shell Editor, open `src/main/java/hello/HelloController.java` and change the return line to `return "Hello, Skaffold!"`.
+
+Switch back to the first terminal and see that its rebuilding and redeploying the app.
+
+Switch back to the third terminal and run the following command to watch until only one pod is running:
+```bash
+watch kubectl get pods
+```
+
+Once there is only one pod running, meaning the latest pod is deployed, see that your app changes are live:
+```bash
+curl http://EXTERNAL_IP:8080
+```
+
+### Updating Buildpacks
+
+Perhaps the best benefit of buildpacks is that it reduces how much work developers need to do to patch their applications if the security team highlights a library vulnerability.
```

Comment:
Can you explain this part a bit? How is the buildpack being updated? Does GCP ship a new version?